corehq/apps/translations/validator.py

```diff
@@ -77,7 +81,7 @@ def _compare_sheet(self, sheet_name, uploaded_rows, for_type):
         :param for_type: type of sheet, module_and_forms, module, form
         :return: diff generated by ghdiff or None
         """
-        columns_to_compare = COLUMNS_TO_COMPARE[for_type] + [self.default_language_column]
+        columns_to_compare = COLUMNS_TO_COMPARE[for_type] + self.lang_cols_to_compare
```

Comment:
Why does this compare both the default language and the selected language? I'd have thought it would only compare the selected language.

Reply:
I believe that default language should always be compared and will be useful. The new language is an additional language to be verified.
I think we can just keep it one since we already set the language to default language in the UI, so that caters for the default behaviour. But i find default language comparison useful to confirm if the default translation changed, which should be flagged even if user has not requested for it.

Reply:
cool, thanks for the explanation